formula/src/main/java/com/instacart/formula/internal/FormulaManagerImpl.kt

```diff
@@ -175,14 +175,14 @@ internal class FormulaManagerImpl<Input, State, Output>(
 
     override fun markAsTerminated() {
         terminated = true
-        frame?.transitionCallbackWrapper?.terminated = true
-        callbacks.disableAll()
+        frame?.transitionDispatcher?.terminated = true
         children?.forEachValue { it.markAsTerminated() }
     }
 
     override fun performTerminationSideEffects() {
         children?.forEachValue { it.performTerminationSideEffects() }
         updateManager.terminate()
+        callbacks.disableAll()
```

Comment:
The reason I'm moving this from `markAsTerminated` to `performTerminationSideEffects` is to allow termination side-effects to execute.

Reply:
This was needed for https://github.com/instacart/formula/pull/245/files#r719839477